test/k8sT/DatapathConfiguration.go

```diff
@@ -912,7 +912,7 @@ func testHostFirewall(kubectl *helpers.Kubectl) {
 
 	demoHostPolicies := helpers.ManifestGet(kubectl.BasePath(), "host-policies.yaml")
 	By(fmt.Sprintf("Applying policies %s", demoHostPolicies))
-	_, err := kubectl.CiliumPolicyAction(randomNs, demoHostPolicies, helpers.KubectlApply, helpers.HelperTimeout)
+	_, err := kubectl.CiliumPolicyAction("", demoHostPolicies, helpers.KubectlApply, helpers.HelperTimeout)
```

Comment:
Would it make sense to have a CCNP-dedicated action wrapping `CiliumPolicyAction` with `""` as default namespace parameter to make the intent clear?
I'm thinking this could also be the right place to document in-code why CCNP applies work that way.

Reply:
I also think this makes sense.

Reply:
There's actually already a `CiliumClusterwidePolicyAction` :facepalm:

Reply:
Yea, even though its code could have been at least in part shared with `CiliumPolicyAction`, but whatever.